src/controllers/firewalls/firewall.controller.ts

```diff
@@ -169,4 +169,27 @@ export class FirewallController extends Controller {
             throw error;
         }
     }
+    @Validate()
+    async installPlugin(req: Request): Promise<ResponseBuilder> {
+        try{
+            const pgp = new PgpHelper(req.session.pgp);       
+            const communication = new AgentCommunication({
+                protocol: req.body.install_protocol,
+                host: req.body.address,
+                port: req.body.install_port,
+                apikey: await pgp.decrypt(req.body.install_apikey)
+            });
+            
+            let data = await communication.installPlugin(req.body.name,req.body.enable);
```

Comment:
Once you define `installPlugin` function return type, then you can define the `data` type properly.

Reply:
Use `const` when possible.